libc/docs/arch_support.rst

```diff
@@ -0,0 +1,22 @@
+Architecture Support
+====================
+
+The currently continuously tested CPU architures are:
+
+* x86_64
+* aarch64
+* arm
+* riscv32
+* riscv64
+
+i386 support is in the works.
+
+The currently continuously tested GPU architures are:
+
+* amdgcn-amd-amdhsa
+* nvptx64-nvidia-cuda
```

Comment:
I'd just call it `amdgpu` and `nvptx` instead of the full triple. Those are the only ones we currently have support for. Now that SPIR-V is being promoted to an official backend I've thought of starting a really limited port for the SPIR-V target, but that's not available yet and would likely require a lot of infrastructure work. (Intel uses SPIR-V as its IR I believe).

Reply:
Also, might not even need to specify GPU / CPU, just say all the supported target architectures that are tested. Technically it's `nvptx64`, but NVIDIA removed support for the 32-bit target quite some time ago.

Reply:
good idea, https://github.com/llvm/llvm-project/pull/120125/commits/5785d5b100c8efd3a07ab719b62803628a2341e6